compendium/ServiceComponent/test/CMakeLists.txt

```diff
@@ -0,0 +1,118 @@
+#-----------------------------------------------------------------------------
+# Build and run the GTest Suite of tests
+#-----------------------------------------------------------------------------
+
+set(us_servicecomponent_test_exe_name usServiceComponentTests)
+
+include_directories(
+  ${GTEST_INCLUDE_DIRS}
+  ${GMOCK_INCLUDE_DIRS}
+  )
+
+
+if (US_COMPILER_CLANG OR US_COMPILER_APPLE_CLANG)
```

Comment:
I don't think this entire if statement is necessary since it exists in the top level CMakeLists.txt

Reply:
Done.